### PR TITLE
Boomkarks UI tests

### PR DESCRIPTION
**Pull request by sv-ohorvath**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt [hunk 1]
```diff
@@ -43,57 +43,155 @@ class BookmarksTest {
     @After
     fun tearDown() {
         mockWebServer.shutdown()
+        val bookmarksStorage = activityTestRule.activity?.bookmarkStorage
```
Comment (npark-mozilla) [inline app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt]: Perhaps a comment saying that the bookmarks are deleted after each test?

File: app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt [hunk 2]
```diff
@@ -43,57 +43,155 @@ class BookmarksTest {
...
+        }
+    }
+
+    @Test
+    fun verifyBookmarkButtonTest() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+        }.openQuickActionBar {
```
Comment (npark-mozilla) [inline app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt]: this is good for now, but I hear the quick action bar will be gone later this year.  We'll need to update the test when it happens.

Comment (sv-ohorvath) [reply app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt]: ok, I'll make sure to keep this up to date

File: app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt (hunk 1 above)
Comment (sv-ohorvath) [reply app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt]: Done, updating the branch right now

**Pull request merged**